nengo/neurons.py

```diff
@@ -171,8 +175,8 @@ def rates(self, x, gain, bias):
 
         Parameters
         ----------
-        x : (n_samples, 1), (n_samples, n_neurons) array_like
-            Scalar inputs for which to calculate current.
+        x : (n_samples,) array_like
```

Comment:
Should this have been kept as `x : (n_samples, 1), (n_samples, n_neurons) array_like` (same as the first version)? Or am I misunderstanding why this is different in this second version?